locales/en/app.ftl

```diff
@@ -79,6 +79,9 @@ compromised-accounts = Compromised accounts:
 # compromised-data = the kind of user data exposed to hackers in data breach.
 compromised-data = Compromised data:
 
+# the kind of user data exposed to hackers in data breach.
```

Comment:
I moved them to the V2 section in 068b0e73f.

I was hesitant about moving them to a new file already, since I'm not sure if that will need alignment with the Pontoon folks, so I left that for now.